R/pkg/R/context.R

```diff
@@ -225,3 +225,26 @@ broadcast <- function(sc, object) {
 setCheckpointDir <- function(sc, dirName) {
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(dirName))))
 }
+
+#' @title Run a function over a list of elements, distributing the computations with Spark.
+#'
+#' @description
+#' Applies a function in a manner that is similar to doParallel or lapply to elements of a list.
+#' The computations are distributed using Spark. It is conceptually the same as the following code:
+#'   unlist(lapply(list, func))
+#'
+#' @param list the list of elements
+#' @param func a function that takes one argument.
+#' @noRd
```

Comment:
if this is an "exported" function then it should not have `@noRd` - please see something like [this](https://github.com/thunterdb/spark/blob/7264/R/pkg/R/SQLContext.R#L73)